src/types/FILTERS.ts

```diff
@@ -0,0 +1,5 @@
+export enum FILTERS {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLITED = 'Completed',
```

Comment:
There's a spelling mistake in the 'COMPLETED' filter. It's currently spelled as 'COMPLITED'.